Source/santad/Logs/EndpointSecurity/Serializers/BasicString.mm

```diff
@@ -895,19 +896,48 @@ static void AppendEventInstigatorOrFallback(std::string &str,
   }
 }
 
+static inline void AppendStringToken(std::string &str, std::string key, es_string_token_t val) {
```

Comment:
It seems like this could be used in lots of other places, are you planning to refactor the existing serializers to use this?

Reply:
Think this would be ideal, yeah.